privacy-and-terms/privacy-policy.md

```diff
@@ -1,122 +1,108 @@
 # Choreo Privacy Policy
 
-Choreo takes your privacy seriously. This privacy policy explains the information we collect when you visit [www.choreo.dev](https://console.choreo.dev/) or [www.choreo.io](https://console.choreo.dev/login/)  and how we treat that information. Choreo is owned by WSO2 Inc (“WSO2”) and whenever the terms “we” “us” or “our” appear in this policy, we’re talking about WSO2 and its subsidiary companies.  When we use the terms “you” and “your” in this policy, we mean the person or entity visiting our site or using the services on it.
+Choreo takes Your privacy seriously. This privacy policy explains the information we collect when You visit [www.choreo.dev](https://console.choreo.dev/) and how we treat that information. Choreo is owned by WSO2 Inc (“WSO2”) and whenever the terms “we” “us” or “our” appear in this policy, we’re talking about WSO2 and its subsidiary companies.  When we use the terms “You” and “Your” in this policy, we mean the person or entity visiting our site or using the services on it.
```

Comment:
Why are we use uppercase `Y` when we refer to `You` and `Your` in the middle of sentences? If we want to highlight it as a legal term shouldn't we use backtick or quotes? Otherwise, this looks like incorrect English. 

https://zapier.com/privacy
https://www.mulesoft.com/privacy-policy-old

Didn't see a  such pattern with any other place as well. @Nashaath

Reply:
I had initially added the content without the capitalization because most of our competitor privacy policies also did not have such a capitalization. However, when I checked with Suranee, she mentioned that the terms need to be capitalized. So I changed it due the legal requirement.

Reply:
I'll check with Suranee again on this.

Reply:
Checked with Suranee and her suggestion was to revert the capitalization after adding the following sentence as a new para:
"When we use the terms “You” and “Your” in this policy, we mean the person or entity visiting our site or using the services on it." 
I've done the suggested change.